modules/acm/src/main/java/edu/gemini/epics/acm/CaObserveSenderImpl.java

```diff
@@ -42,7 +42,7 @@
     private final short MRK_PRESET = 2;
     private final short MRK_IDLE = 0;
 
-    private final Boolean trace = false;
+    private final Boolean trace = true;
```

Comment:
Maybe we should make this configurable to enable/disable more easily
I understand it is deep in the hirerachy but what about using a sys property
`trace = System.getProperty('epics.observe.trace') != null`?